### ISSUE TITLE
Learning rate decaying schedule.

### ISSUE THREAD

**Issue by Blurryface0814** [PRBonn/point-cloud-prediction#8]
Hello authors, 

Thanks for your nice work!
Could you please tell me details about the learning rate decaying schedule for trainning. I cannot reproduce the results in your paper.

**Comment by benemer** [comment]
Hi @Blurryface0814,

Thank you for your interest in our work. Please check the scheduler we use in our code [here](https://github.com/PRBonn/point-cloud-prediction/blob/833ce3f12f18e6060ea69b5e9a9f8f9cf517c73d/pcf/models/base.py#L67). Please let me know if anything is unclear. If you just want to reproduce the results, you can also skip training the models yourself and use our pre-trained ones.

Best regards
Benedikt

**Comment by Blurryface0814** [comment]
> Hi @Blurryface0814,
> 
> Thank you for your interest in our work. Please check the scheduler we use in our code [here](https://github.com/PRBonn/point-cloud-prediction/blob/833ce3f12f18e6060ea69b5e9a9f8f9cf517c73d/pcf/models/base.py#L67). Please let me know if anything is unclear. If you just want to reproduce the results, you can also skip training the models yourself and use our pre-trained ones.
> 
> Best regards Benedikt

Thank you!
In "parameters.yml", I set LR to 0.001, LR_EPOCH ot 1 ,LR_DECAY to 0.5 and LOSS_WEIGHT_CHAMFER_DISTANCE to 0.0, but I found that after about 10 epochs, the loss of the model on the training set decreased very slowly, and the model also converged on the verification set. At this time, the loss on the verification set is about 1.2, and it has not decreased in the following epochs.
Did I set LR unreasonably? Could you please tell me your LR and LR_ DECAY settings? Thanks!

**Comment by benemer** [comment]
I think your learning rate decay is too strong since the learning rate gets reduced by half after each epoch using this setting. Note that in the paper, we multiply the learning rate by 0.99 after each epoch instead of 0.5.

You can find our settings in the [parameters.yml](https://github.com/PRBonn/point-cloud-prediction/blob/main/config/parameters.yml):
![image](https://user-images.githubusercontent.com/38326482/196174698-54c187fe-a67c-4040-9f7c-9ffae51b9e97.png)


Please check Pytorch's `StepLR` [documentation](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.StepLR.html) for further information on the scheduler.

**Comment by Blurryface0814** [comment]
> I think your learning rate decay is too strong since the learning rate gets reduced by half after each epoch using this setting. Note that in the paper, we multiply the learning rate by 0.99 after each epoch instead of 0.5.
> 
> You can find our settings in the [parameters.yml](https://github.com/PRBonn/point-cloud-prediction/blob/main/config/parameters.yml): ![image](https://user-images.githubusercontent.com/38326482/196174698-54c187fe-a67c-4040-9f7c-9ffae51b9e97.png)
> 
> Please check Pytorch's `StepLR` [documentation](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.StepLR.html) for further information on the scheduler.

Thank you very much! I will use these settings to train the model.
